docker-compose.yml

```diff
@@ -73,7 +71,3 @@ networks:
         driver: bridge
     redis:
         driver: bridge
-
-volumes:
-    postgres:
-        driver: local
```

Comment:
with removing this we loose all data weekly on image update, this is not very comfortable. Pls let the named volume stay it is a very nice feature I use this everytime as possible

Reply:
73b959b8cb63d55cded4bf09e170c3fdc22a536f